test/artefacts/benefit-cap-calculator/default.html

```diff
@@ -78,13 +78,13 @@ <h3 class="previous-answers-title">
         <table>
           <tbody>
               <tr class="section">
-  <td class="previous-question-title">The benefit cap is changing.</td>
+  <td class="previous-question-title">Which period do you want to check for?</td>
     <td class="previous-question-body">
-    The current benefit cap</td>
+    Now</td>
 
   <td class="link-right">
     <a href="/benefit-cap-calculator/y?previous_response=default">
-      Change<span class="visuallyhidden"> answer to "The benefit cap is changing."</span>
+      Change<span class="visuallyhidden"> answer to "Which period do you want to check for?"</span>
```

Comment:
I agree and did consider making that change. I decided against it partly due to time available and partly as I wanted to reuse as many commits as possible from PR #2542.